loopchain/utils/message_queue/stub_collection.py

```diff
@@ -13,18 +13,26 @@
 # limitations under the License.
 
 import logging
-
 from typing import Dict, TYPE_CHECKING
+
 from loopchain.components import SingletonMetaClass
 
 if TYPE_CHECKING:
     from loopchain.peer import PeerInnerStub
-    from loopchain.channel.channel_inner_service import ChannelInnerStub, \
-        ChannelTxReceiverInnerStub, ChannelTxCreatorInnerStub
-    from loopchain.scoreservice import ScoreInnerStub, IconScoreInnerStub
+    from loopchain.channel.channel_inner_service import (ChannelInnerStub,
+                                                         ChannelTxCreatorInnerStub,
+                                                         ChannelTxReceiverInnerStub)
+    from loopchain.scoreservice import IconScoreInnerStub
 
 
 class StubCollection(metaclass=SingletonMetaClass):
+    """
+    rpc stubs with rabbitmq for inter process communication
+
+    FIXME : consider singleton to borg
+    ref : https://github.com/faif/python-patterns/blob/master/patterns/creational/borg.py
```

Comment:
How about putting StubCollection as an attribute in Communication class instead of singleton?